### DISCUSSION TITLE
Does battery charges when the capture is controlled by USB?

### DISCUSSION THREAD

**Discussion by oreixa** [gopro/labs#397] [Q&A] [answered]
Hi

I have this code to start recording when power on usb is detected:

!MBOOT="!LRecOnUsb"!SAVERecOnUsb=>u!S~!10E!R

Does the battery charges when the script is running? Screen icon don't seem to be active.

If the battery don't charge how can I modify the script to shutdown after stoping the recording?

Many thanks

**Comment by dnewman-gpsw** [comment, marked as answer]
Battery does not charge while recording. >u!S~!10E+!1O!R. The will shutdown after the record ends.

**Comment by oreixa** [comment]
Thanks
